module2/projects/little_shop/index.md

```diff
@@ -0,0 +1,112 @@
+
+Your job is to expose the data that powers the site through an API that a front end will consume. You will be working with an existing front end application that consumes these APIs, and part of your work will also be to make a few adjustments to that front end as well (see "Connecting to the Front End" below). 
+
+## Learning Goals
+
+### Explicit
+
+* Use ActiveRecord and SQL to write queries that deal with one-to-many database relationships
+* Expose API endpoints to CRUD database resources
+* Validate models and handle sad paths for invalid data input
+* Test both happy and sad path functionality based on JSON contracts
+* Use MVC to organize code effectively, lmiting data logic in controllers and serializers
+* Track user stories with GitHub Projects
+* Deploy your application to the internet
+* Improve an existing FE application by:
+    * Styling the user interface
+    * Refactoring JavaScript code
+    * Adding an additional FE feature
+
+### Implicit
+Below are skills that are more general/necessary on the job that are practiced by completing the goals above.
+
+* Reading Documentation
+* Time Management
+* Prioritizing Work
+* Breaking down large project into small pieces
+* Breaking down a problem into small steps
+* Practice individual research (articles, videos, mentors)
+
+<section class="dropdown">
+## Project Management
+
+Groups must utilize a project board in order to organize areas of work and allow for  visibility of each other's progress. In addition, groups will be evaluated on the evidence of meaningful code review, either via comments on a PR, or through discussion of live code review during their evaluation. 
+</section>
+
+<section class="dropdown">
+### Postman
+
+By importing the JSON files noted below, you can open the Postman "Runner" and execute a large suite of tests.
+
+**Note: not ALL of the Postman tests will be passing in order to pass the project**. You will need to examine the tests you NEED to run, since we're giving students some choice in how to build the application.
+
+#### Importing the Test Suite
+
+Download the test suites for Postman:
+* [Little Shop BE, Section 1](./LittleShopBE,Part1.postman_collection.json)
+* [Little Shop BE, Section 2](./LittleShopBE,Part2.postman_collection.json)
+
+Click on each link to load it in your browser, then hit Cmd-S to save it to your system.
+
+In Postman, in the top left corner, click on the "Import" button, and use the file selector to locate the two files on your operating system.
+
+Next, you'll "confirm" the import. The test suite should display as a "Postman Collection v2.1" and import as a "Collection". Click the "Import" button to continue.
+
+Within your collections in Postman, you should see two collections-- "Rails Engine Lite, Part 1", and "Rails Engine Lite, Part 2". 
```

Comment:
My collections are called "Little Shop BE, Part 1" and "Little Shop BE, Part 2".  Not sure if I did something weird, or if we should update these names 😆

Reply:
oops I updated them everywhere except here! they should not be called Rails Engine anymore